src/main/host/thread_preload.c

```diff
@@ -143,6 +142,11 @@ static pid_t _threadpreload_fork_exec(ThreadPreload* thread, const char* file, c
         case 0: {
             // child
 
+            // Set the working directory
+            if (workingDir != NULL && chdir(workingDir) < 0) {
```

Comment:
Probably doesn't make sense to silently ignore a NULL workingDir, since that'd be a programming error. Maybe `utility_assert(workingDir)`?

Reply:
Done